src/main/java/frc/robot/vision/VisionFilters.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.vision;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.constants.Field;
+import frc.robot.vision.data.AprilTagVisionData;
+import frc.robot.vision.data.VisionData;
+import frc.utils.Filter;
+import frc.utils.ToleranceUtils;
+
+public class VisionFilters {
+
+	public static Filter<VisionData> isPitchInTolerance(Rotation2d pitchTolerance, Rotation2d wantedPitch) {
```

Comment:
In tolerance compared to what? same for isRollInTolerance, isAprilTagInTolerance.
rename function